src/current/cockroachcloud/costs.md

```diff
@@ -171,11 +166,7 @@ For CockroachDB {{ site.data.products.basic }}, data transfer is included in the
 
 <section class="filter-content" markdown="1" data-scope="standard">
 
-{{site.data.alerts.callout_info}}
-Customers will not be charged for data transfer for CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} clusters during the current [Preview](#usage-based-billing-metrics-in-preview) of usage-based metrics.
-{{site.data.alerts.end}}
-
-For CockroachDB Standard and Advanced, CockroachDB Cloud bills directly to customers based on their usage with a zero-dollar rate during the preview period.
+For CockroachDB Standard and Advanced, CockroachDB Cloud bills directly to customers based on their usage.
```

Comment:
It's confusing that we mention both Standard and Advanced in both the tabs for Standard and Advanced. This sentence makes me immediately wonder about Basic. I don't think it would if we mention on the tier that is appropriate to the tab.